modules/configure-policy-enforcement-creating-policies.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage_security_policies/custom-security-policies.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="configure-policy-enforcement-creating-policies_{context}"]
+= Configuring policy enforcement
+
```

Comment:
Not new content, was broken out from the original file. Will be updated in a separate PR.